Formula/par2.rb

```diff
@@ -13,39 +13,55 @@ class Par2 < Formula
     sha256 "367db0a915a8bdaa9de30be2abc6de9e641d8031864df71998efa7d1be7ef53f" => :mountain_lion
   end
 
-  # Clang doesn't like variable length arrays of non-POD types.
-  patch :DATA
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
 
-  # Fixes compilation with GCC 4 and still required for clang
-  patch do
-    url "https://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/app-arch/par2cmdline/files/par2cmdline-0.4-gcc4.patch?revision=1.1"
-    sha256 "eda0a381f944b1bc9d3d78bf4526f77620bcb01de48abcb08878178e47c833f7"
-  end
-
-  # http://parchive.cvs.sourceforge.net/viewvc/parchive/par2-cmdline/par2creatorsourcefile.cpp?r1=1.4&r2=1.5
-  patch :p0 do
-    url "https://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/app-arch/par2cmdline/files/par2cmdline-0.4-offset.patch?revision=1.2"
-    sha256 "c4820b11376c9932ece944752ddd388fb50fcbcd47aaadda073997142952d969"
-  end
+  # The tests for this version of par2cmdline use the `iflag` option to dd,
+  # which doesn't exist in OS X. `iflag` has been removed as of commit
+  # 855f3096c9f52cd0fbdb9e237c6c9624f7b90ea6, but no release has been cut since
```

Comment:
This patch isn't available on the web; I generated it locally to make the tests pass. It will also disappear as of the next release of par2cmdline, since the problem has already been fixed upstream; they just haven't a cut a release yet. Do you mind leaving it in as it is, since it's temporary and there's no obvious place for me to host it, being an unofficial patch?